### ISSUE TITLE
Show server info somewhere

### ISSUE THREAD

**Issue by nikclayton** [tuskyapp/Tusky#3970]
Servers provide information about posting limits and other info at `/api/v2/instance`.

Some of that information could be useful to bundle up and display on a screen to the user somewhere. Maybe on the "About" screen (might be a bit non-discoverable), maybe as another tab on their profile screen (makes some sense, since the profile is tied to a specific server).

Maybe as account preferences? They're not user-settable preferences, but maybe that's the most discoverable place?

Useful information to highlight could be:

- max_featured_tags
- max_characters
- max_media_attachments
- supported_mime_types
- image_size_limit
- image_matrix_limit (rendered as useful maximum width/height)
- video_size_limit
- video_matrix_limit (ditto)
- polls > max_options
- polls > max_characters_per_option
- polls > max_expiration
- translation > enabled
- contact info
- server rules

* * * *
- Tusky Version: 23.0

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by sanao1006** [comment]
Hello. I would like to work on this assignment!

**Comment by sanao1006** [comment]
I would like to work on this task!

**Comment by Tak** [comment]
Hi @sanao1006 - feel free to pick this up, and don't hesitate to [connect with us on matrix](https://matrix.to/#/#Tusky-General:matrix.org) if you need help / want feedback / etc.

**Comment by sanao1006** [comment]
Hi! I have a question.

If I use the `/api/v2/instance` endpoint response instead of `/api/v1/instance`, do I need to add a new function to MastodonApi to receive the response for that endpoint and create a new data class to store the response?  (e.g. `getServerInfo()` function and `ServerInfo` data class)

**Comment by Tak** [comment]
Yes, you'll need a new function in `MastodonApi`, and a new data type for the result (unless it's similar enough to the v1 result that it can just be expanded)

**Comment by sanao1006** [comment]
> Yes, you'll need a new function in `MastodonApi`, and a new data type for the result (unless it's similar enough to the v1 result that it can just be expanded)

Is it okay if I create a PR once I've implemented this stage of the project, since there may be a lot of commits for the issue?

**Comment by sanao1006** [comment]
@Tak 
Sorry, I'm having trouble finding time to work on this task, so I'll give up...

**Comment by Tak** [comment]
Completely understandable, thanks for your effort!

**Comment by strk** [comment]
I was also looking for this feature. An About Instance menu would be useful. I wanted to know the username of my instance admin and could not find it
